sd-decrypt/decrypt-sd-submission

```diff
@@ -51,6 +52,17 @@ for z in zips:
         zf.extractall(tmpdir + "/extracted/")
     os.unlink(z)
 
+gpgs = glob.glob(tmpdir + "/*gpg")
+for g in gpgs:
+    match = re.search('\d-(.*)-(msg|doc).*$', g)
+    source = match.group(1)
+    target = "{}/extracted/{}".format(tmpdir, source)
+
+    if not os.path.exists(target):
+        os.makedirs(target)
+
+    os.rename(g, "{}/{}".format(target, os.path.basename(g)))
```

Comment:
It would be nicer to read if we have `os.path.join` here too.

Reply:
👍